src/main.cpp

```diff
@@ -108,6 +109,7 @@ int main(int argc, char *argv[])
         "-unload %s",    &unloadFiles,   "Remote: unload loaded files matching the given (unix shell style) pattern",
         "-quit",         &quitRemote,    "Remote: close the existing displaz window",
         "-add",          &addFiles,      "Remote: add files to currently open set, instead of replacing those with duplicate labels",
+        "-modify",       &mutateData,    "Remote: mutate data already loaded with the matching label (requires displaz .ply with an \"index\" field to indicate mutated points)",
```

Comment:
There's not a huge amount in it... and it's going to be used almost entirely from scripts anyway, so let's go with what you have.